src/App.css

```
@@ -0,0 +1,24 @@
+.container{
+  --lightBlue:rgb(55, 168, 225);
+  --white:white;
```

Եթե ամբողջ կայքում օգտագործվող փոփոխականներ են, արժի տեղափոխել իրենց `:root { }` ի մեջ, ու նաև կարելի է տանել `index.css`

```suggestion
:root {
  --light-blue: rgb(55, 168, 225);
  --white: white;
```


Փոքր նշումներ՝ 
1. CSS-ում ընդունված անունները դեշերովն են `lightBlue` -> `light-blue`

---

More about `:root` selector [here.](https://www.digitalocean.com/community/tutorials/css-root-pseudo-class)